src/objects.c

```diff
@@ -675,7 +675,7 @@ TOOL("leash",               None, 1, 0, 0, 0, 65, 12, 20, LEATHER, HI_LEATHER),
 TOOL("stethoscope",         None, 1, 0, 0, 0, 25,  4, 75, IRON, HI_METAL),
 TOOL("tinning kit",         None, 1, 0, 0, 1, 15,100, 30, IRON, HI_METAL),
 TOOL("tin opener",          None, 1, 0, 0, 0, 35,  4, 30, IRON, HI_METAL),
-TOOL("can of grease",       None, 1, 0, 0, 1, 15, 15, 20, IRON, HI_METAL),
+TOOL("can of grease",       None, 1, 0, 0, 1, 15, 15, 20, PLASTIC, CLR_BRIGHT_BLUE),
```

Comment:
Well, maybe you're not old enough to have seen metallic cans of grease. :-)

https://images.ehive.com/accounts/5386/objects/images/r5l8pe_a36v_m.jpg

Plastic are somewhat anachronistic for NetHack but then again, NetHack is in itself anachronistic.